projects/avalon-finance/index.js

```diff
@@ -74,7 +75,14 @@ const listaMarkets = {
 	bsc: aaveExports('', '', undefined, ['0x5157f63bE7808DEB090Eee7762e917745896A09E'], { v3: true }), // BSC - ListaDAO
 }
 
+// @note USDa Defi Lending Markets, not USDaLend
+const usdaDefiLendingMarkets = {
+	iotex: aaveExports('', '', undefined, ['0xDB52DD393e3a5e95d3B7C7e1C42cC06bb807A369'], { v3: true }),
```

Comment:
@jasmin-avalon I meant this line

Reply:
Oh, the IoTex-USDa does a duplicate one. I've removed it in the latest commit(https://github.com/DefiLlama/DefiLlama-Adapters/pull/12465/commits/55ad0917a34c8f896fd31fc8b409cae48f58a14d). 
We just want to add the Zetachain-PumpBTC and Taiko-USDa to DeFi Lending.  
I think it should be good now.